### DISCUSSION TITLE
Sweave vignette

### DISCUSSION THREAD

**Discussion by anikoszabo** [r-hub/rhub#636] [Q&A]
I am doing a minor update to my package, and wanted to check it with the new rhub. Unfortunately, all checks failed
[https://github.com/anikoszabo/CorrBin/actions/runs/10618750115](https://github.com/anikoszabo/CorrBin/actions/runs/10618750115)  apparently because there is no latex on the servers. I have a .Rnw file in the vignettes folder, and I can build it locally just fine.

![image](https://github.com/user-attachments/assets/a3066e8d-16f3-4b73-988b-dd1bf2b50b12)

I am not sure what is the correct workaround. I can compile the file into PDF, but where do I put it? I also don't want to just keep the static file locally. I also want to avoid converting it to Rmd just to be able to run checks.

**Comment by gaborcsardi** [comment]
There is no workaround for this currently, apart from ignoring the failures.

**Reply by anikoszabo** [reply]
Thanks for the quick response. I will hold off using this tool for this package, but will test it on others

**Reply by gaborcsardi** [reply]
Lots of R-hub containers do not build the vignettes, anyway. If the vignette builds on your machine, then it does not really make too much sense to test it elsewhere.

**Reply by anikoszabo** [reply]
I am happy not to test the vignette building. So to refine my question: how do I avoid testing the vignette building process on R-hub, while keeping the Rnw file around and building the vignette locally (and placing it somewhere in the package directory)?

**Comment by ahoundetoungan** [comment]
Is there any solution for this issue? So we can't check packages using Rhub when we have a vignette?
